MAINTENANCE.md

```diff
@@ -135,6 +137,22 @@ The actual script `package.json` is used by Vercel to install the necessary pack
 build the documentation, copy the files to a 'public' folder and deploy that to the web,
 see https://vercel.com/docs/build-step.
 
+
+## Dependencies Policy
+
+PyGMT has adopted [NEP29](https://numpy.org/neps/nep-0029-deprecation_policy)
+alongside the rest of the Scientific Python ecosystem, and therefore supports:
+
+* All minor versions of Python released 42 months prior to the project,
+  and at minimum the two latest minor versions.
+* All minor versions of NumPy released in the 24 months prior to the project,
+  and at minimum the last three minor versions.
+
+In `setup.py`, the `python_requires` variable should be set to the minimum
+supported version of Python. Minimum Python and NumPy version support should be
+adjusted upward on every major and minor release, but never on a patch release.
```

Comment:
I didn't add the line "All supported minor versions of Python should be in the test matrix and have binary artifacts built for the release." as in https://numpy.org/neps/nep-0029-deprecation_policy#implementation because:

1. We're not testing the middle Python version (Python 3.8)
2. We're using no-arch builds on conda-forge and a single [PyGMT Python 3 wheel on PyPI](https://pypi.org/project/pygmt/0.3.1/#files), instead of having specific binary artifacts for each Python version.

Reply:
Sounds good to me.